docs/fuzz-targets.md

```diff
@@ -0,0 +1,171 @@
+# Fuzzing using fuzz targets and the CLI
+
+Creating fuzz targets and executing those via CLI commands is straight forward
```

Comment:
- straightforward.
- nitpick: Copy paste of paragraph

Reply:
done
True, but just the initial sentence :grin: